<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1681094916588' stop='1681094987230'>
    <name>Feature Perform an Order</name>
    <title>Feature Perform an Order</title>
    <test-cases>
        <test-case start='1681094936594' status='failed' stop='1681094946605'>
            <name>Add an item</name>
            <title>Add an item</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within the 20000 ms timeout specified by jest.setTimeout.</message>
                <stack-trace>Timeout - Async callback was not invoked within the 20000 ms timeout specified by jest.setTimeout.Error: Timeout - Async callback was not invoked within the 20000 ms timeout specified by jest.setTimeout.
                    at mapper (C:\Users\eduar\Documents\GitHub\eduardotuesdayend\node_modules\jest-jasmine2\build\queueRunner.js:21:45)
                    at C:\Users\eduar\Documents\GitHub\eduardotuesdayend\node_modules\jest-jasmine2\build\queueRunner.js:62:41
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1681094946605' status='failed' stop='1681094956612'>
            <name>Purchase</name>
            <title>Purchase</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within the 20000 ms timeout specified by jest.setTimeout.</message>
                <stack-trace>Timeout - Async callback was not invoked within the 20000 ms timeout specified by jest.setTimeout.Error: Timeout - Async callback was not invoked within the 20000 ms timeout specified by jest.setTimeout.
                    at mapper (C:\Users\eduar\Documents\GitHub\eduardotuesdayend\node_modules\jest-jasmine2\build\queueRunner.js:21:45)
                    at C:\Users\eduar\Documents\GitHub\eduardotuesdayend\node_modules\jest-jasmine2\build\queueRunner.js:62:41
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1681094956612' status='failed' stop='1681094986621'>
            <name>Place an Order</name>
            <title>Place an Order</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within the 20000 ms timeout specified by jest.setTimeout.</message>
                <stack-trace>Timeout - Async callback was not invoked within the 20000 ms timeout specified by jest.setTimeout.Error: Timeout - Async callback was not invoked within the 20000 ms timeout specified by jest.setTimeout.
                    at mapper (C:\Users\eduar\Documents\GitHub\eduardotuesdayend\node_modules\jest-jasmine2\build\queueRunner.js:21:45)
                    at C:\Users\eduar\Documents\GitHub\eduardotuesdayend\node_modules\jest-jasmine2\build\queueRunner.js:62:41
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>